master/buildbot/status/buildstep.py

```diff
@@ -396,13 +408,31 @@ def asDict(self, request=None):
             self.build.builder.status.getURLForThing(l) + args]
                 for l in self.getLogs()]
 
-        result["urls"] = self.getURLs()
-
+        result["urls"] = self.temporaryUrlConversion()
```

Comment:
I don't want to make that change until the UI is actually ready to support it, or I won't be able to test the changes I make in Katana.